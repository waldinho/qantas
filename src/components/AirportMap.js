import React, {Component} from 'react';
import styled from 'styled-components';
import {Map, GoogleApiWrapper} from 'google-maps-react'

const Wrapper = styled.div`
`;

// const mapOptions = {
//     styles: mapStyles
// };

class AirportMap extends Component {
    render() {
        const {
            google,
            lat,
            long,
        } = this.props
        return (
            <Wrapper>
                <Map 
                    google={google}
                    zoom={8}
                    // options={mapOptions}
                    initialCenter={{
                        lat: lat,
                        lng: long
                    }}
                    style={{
                        margin: '1.75rem 0 0 0',
                        // height: '420px',
                    }}>
                </Map>
            </Wrapper>
        );
    }
}

export default GoogleApiWrapper({
    apiKey: ('AIzaSyCksyb1zfYmQHK-Zb69o1Hw0YWbKpPHXbU')
})(AirportMap);