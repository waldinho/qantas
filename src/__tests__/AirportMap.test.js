import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import AirportMap from '../components/AirportMap';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<AirportMap />, div);
  ReactDOM.unmountComponentAtNode(div);
});

describe('<AirportMap /> with no props', () => {
  const container = shallow(<AirportMap />);
  it('should match the snapshot', () => {
    expect(container.html()).toMatchSnapshot();
  })
})
