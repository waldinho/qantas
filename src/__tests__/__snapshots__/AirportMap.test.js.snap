// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AirportMap /> with no props should match the snapshot 1`] = `"<div>Loading...</div>"`;
